main/auth/github.py

```diff
@@ -48,9 +48,9 @@ def retrieve_user_from_github(response):
   auth_id = 'github_%s' % str(response['id'])
   user_db = model.User.get_by('auth_ids', auth_id)
   return user_db or auth.create_user_db(
-    auth_id,
-    response.get('name', response.get('login')),
-    response.get('login'),
-    response.get('email', ''),
+    auth_id=auth_id,
+    name=response.get('name', '') or response.get('login'),
```

Comment:
faster? better? more consistent with other code?
(I'm fine with the suggestion, would just like to know your reasoning)